api/src/main/java/com/epam/pipeline/manager/datastorage/DataStorageManager.java

```diff
@@ -343,6 +359,47 @@ public AbstractDataStorage updatePolicy(DataStorageVO dataStorageVO) {
         return createdStorage;
     }
 
+    @Transactional(propagation = Propagation.REQUIRED)
+    public Optional<AbstractDataStorage> createDefaultStorageForUser(final String userName) {
+        final DataStorageTemplate dataStorageTemplate =
+            Optional.ofNullable(preferenceManager
```

Comment:
This parsing doesn't work: parsing of object preference to string throws error. In order to get plain sting value you can use `preferenceManager.getSystemPreference(SystemPreferences.DEFAULT_USER_DATA_STORAGE_TEMPLATE).getValue()`.
Please, check that tests are working properly for this case.